contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/actions.js

```diff
@@ -0,0 +1,83 @@
+import { contentNodeLastSavedState } from './index';
+import { isTempId } from 'shared/utils';
+import client from 'shared/client';
+
+export function loadSummaryContentNodes(context, params) {
+  return client.get(window.Urls['summarycontentnode-list'](), { params }).then(response => {
+    const contentNodes = response.data;
+    context.commit('ADD_CONTENTNODES', contentNodes);
+    return contentNodes;
+  });
+}
+
+export function loadSummaryContentNode(context, id) {
+  return client
+    .get(window.Urls['summarycontentnode-detail'](id))
+    .then(response => {
+      const contentNode = response.data;
+      context.commit('ADD_CONTENTNODE', contentNode);
+      return contentNode;
+    })
+    .catch(() => {
+      return;
+    });
+}
+
+export function loadContentNodes(context, params) {
+  return client.get(window.Urls['contentnode-list'](), { params }).then(response => {
+    const contentNodes = response.data;
+    context.commit('ADD_CONTENTNODES', contentNodes);
+    return contentNodes;
+  });
+}
+
+export function loadContentNode(context, id) {
+  return client
+    .get(window.Urls['contentnode-detail'](id))
+    .then(response => {
+      const contentNode = response.data;
+      context.commit('ADD_CONTENTNODE', contentNode);
+      return contentNode;
+    })
+    .catch(() => {
+      return;
+    });
+}
+
+/* CONTENTNODE EDITOR ACTIONS */
+export function saveContentNode(context, contentNodeId) {
```

Comment:
Is there a way to save multiple nodes with one server call to take advantage of bulk saving?

Reply:
It is definitely possible - although we would want to be doing bulk PATCHes to take advantage of the diffing that we are doing here.